docker-compose.kafka-queue.yml

```diff
@@ -0,0 +1,189 @@
+version: "3.2"
+services:
```

Comment:
I've looked at this config for the first time, it's huge. Is there a way of putting it in another file? We'd also need this in the Kubernetes YAML format in the faas-netes repo.